assets/home/sections/get-help.js

```diff
@@ -1,22 +1,95 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Icon, help, lock, external } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import Section from '../section';
 
+const Item = ( { title, url, extraLink } ) => {
```

Comment:
It would be nice to add some JSDoc to these components. WDYT?

Reply:
Done in 58246cd20